034_reverseAnumber.js

```diff
@@ -0,0 +1,12 @@
+// 34. Reverse a given number
+const reverse = (number) => {
```

Comment:
logic issue: the number should be read in as Integer and use modulus operations to find the reverse
Do not use string methods to reverse a number